src/PasswordHasher/PasswordHasherTrait.php

```diff
@@ -33,8 +33,11 @@ public function passwordHasher()
         }
 
         $passwordHasher = $this->_config['passwordHasher'];
+        if (!$passwordHasher instanceof PasswordHasherInterface) {
+            $passwordHasher = PasswordHasherFactory::build($passwordHasher);
```

Comment:
Isnt the factory taking 2 args? class and config?

The fallback hasher needs config for example. I would enforce passing an object or check if an array was passed that can be used for class and options

And while being there - how about adding a getter and setter and deprecate this method? Or directly remove it. :)